packages/transport-webrtc/test/certificate-persistent.spec.ts

```diff
@@ -0,0 +1,176 @@
+/**
```

Comment:
Because this test imports `libp2p` and other modules, it should be moved to `/packages/integration-tests` in this monorepo.

Only unit tests should be added here.  Obviously the definition is hazy but if you're starting a libp2p node or having to jump through hoops to import other modules from the monorepo, it's probably an integration test.